opal/templates/dask-gateway-application.yaml

```diff
@@ -0,0 +1,92 @@
+{{- if .Values.daskGateway.enabled }}
+{{- $global := .Values }}
+{{- $values := .Values.daskGateway }}
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: dask-gateway
+  # You'll usually want to add your resources to the argocd namespace.
+  namespace: argocd
```

Comment:
namespace should not be listed as argocd should be from a variable. argocd is not guaranteed.